NewDawn/Podfile.lock

```diff
@@ -49,4 +49,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 5d604014fba29e2fcec33932f84a40c76f054150
 
-COCOAPODS: 1.6.1
+COCOAPODS: 1.5.3
```

Comment:
Why is this?

Reply:
This is automatically adjusted when I upgrade the cocoapods (not sure why it goes down)
After this is merged you might want to try if you can setup everything correctly (including running pod install again)